pkg/server/registry/credentials/strategy_validate.go

```diff
@@ -0,0 +1,29 @@
+package credentials
+
+import (
+	"context"
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/v1/remote/transport"
+	"net/http"
+)
+
+// credentialValidate takes a username, password and serverAddress string to validate
+// whether their combination is valid and will succeed login for pushes/pulls.
+func (s *Strategy) credentialValidate(ctx context.Context, username, password, serverAddress string) error {
+	// Build a registry struct for the host
+	reg, err := name.NewRegistry(serverAddress)
+	if err != nil {
+		return err
+	}
+
+	// Build a new transport for the registry which validates authentication
+	scopes := []string{transport.PullScope, transport.PushScope}
```

Comment:
so there's a separate bug where attempting to log in against quay.io fails, because it doesnt like these scopes. Turns out, the scopes arent needed and we can successfully validate without them. can you remove this line and pass `nil` into line 23 instead of `scopes`?